sdk/tests/deqp/functional/gles3/es3fFboTestCase.js

```diff
@@ -135,7 +135,7 @@ var DE_ASSERT = function(x) {
             /** @const @type {goog.NumberArray} */ var supportedSampleCounts = es3fFboTestCase.querySampleCounts(sizedFormat);
             var supported = Array.prototype.slice.call(supportedSampleCounts);
             if (supported.indexOf(numSamples) == -1) {
-                if (numSamples >= 2) {
+                if (minSampleCount == 0 || numSamples > gl.getParameter(gl.MAX_SAMPLES)) {
```

Comment:
OK, I found the spec text to support what you said.

"Implementations must support creation of renderbuffers in these required formats with up to the value of MAX_SAMPLES multisamples, with the exception of signed and unsigned integer formats."

With EXT_color_buffer_float, the last part became "with the exception of signed and unsigned integer, RGBA16F, R32F, RG32F and RGBA32F formats."